en/tutorials-and-examples/blog/part-two.rst

```diff
@@ -261,7 +262,8 @@ ArticlesController::
 
         public function index()
         {
-            $this->set('articles', $this->Articles->find('all'));
+            $articles = $this->set('articles', $this->Articles->find('all'));
```

Comment:
@miguelarcher As soon as your PR is open, and using the Github online editor (as you seems to), everytime you add a commit it will automatically update the PR.

What I'm saying here is that the correct code is
```php
$articles = $this->Articles->find('all');
$this->set(compact('articles'));
````

and your pull request (PR) still contain
```php
$articles = $this->set('articles', $this->Articles->find('all'));
$this->set(compact('articles'));
```
instead.
The correct procedure for would be to go on your github acount, select the branch called `patch-1` and add a commit there to correct the code. If you're using git locally, once you've done your commit you need to use the `git push` command.
You can find information on how to use git locally if you need/are interested into it : http://book.cakephp.org/3.0/en/contributing/code.html#initial-setup

Thank's again for helping. If you can't make it, let us know and we can correct after merging but it's probably more interesting for you to get it right and continue contributing :smile: